cql3/statements/batch_statement.cc

```diff
@@ -24,6 +24,11 @@
 #include <boost/algorithm/cxx11/all_of.hpp>
 #include <boost/range/adaptor/uniqued.hpp>
 
+namespace bo = BOOST_OUTCOME_V2_NAMESPACE;
```

Comment:
The more I see this "bo" thing the less I like it. As I noted above, I think even if we decided to use boost::outcome we should consider hiding it in our own <util/outcome.hh> which will allow us to avoid this weird boilerplate definitions.

Reply:
By the way, as far as I can tell (?) you aren't even using this "bo" namespace below, so it's not even needed.

Reply:
I'm using the convention that we already use for boost::intrusive:

```c++
using bi = boost::intrusive;
```

I'll remove the `using bo = ...` from this file.

Reply:
Oh, I think I parsed your second comment but not the first one. I'll move the `namespace bo` alias into the "utils/result.hh" header.

Reply:
Should we really have namespace aliases in headers? Seems translation unit local to me...

Reply:
@elcallio @nyh I'm a bit confused, because to me it looks like your suggestions oppose each other. @nyh, perhaps I misinterpreted your comments - could you clarify what you would like me to do here?

Reply:
I didn't like the fact that we need to repeat this namespace command (which is particularly ugly in this case, with an ugly-looking macro) in a gazallion place. I agree with @elcallio that maybe calling it with the extremely short name "bo" in a header file is a bit too much. Perhaps a better idea is to hide the fact we are using Boost completely, in our own names - call it util::result and so on, and call the Boost stuff but later we could call std:: stuff when it's standardized, or implement our own.

Anyway, my comment here was just a minor nitpick, prompted by the ugly macro BOOST_OUTCOME_V2_NAMESPACE. I won't insist that you change anything here. It's not important enough to insist here.

Reply:
@nyh Thanks, I understand now. Outside "utils/result.hh", I think we mainly need the `bo` namespace for bo::success and bo::failure, so we could hide then as utils::result_success and utils::result_failure, respectively. For the time being, I would like to prioritize other work related to the "result migration", but after I'll no longer have conflicting work it will be a good idea to hide the `bo` namespace.